packages/cli/src/commands/serve.ts

```diff
@@ -20,6 +20,7 @@ export default class Serve extends Command {
       char: 'm',
       description: 'Turn global mocking on or off',
       default: true,
+      required: true,
```

Comment:
I don't like mixing stuff in a PR very much. I would limit this PR to packaging only - this looks like a feature.

Reply:
Yeah it is, it was to make sure the argument was correctly displayed in the CLI; in any case this is ditched in #243 so when rebasing this will fade away :)

Reply:
:+1: